elytron/src/main/java/org/wildfly/extension/elytron/RealmParser.java

```diff
@@ -92,7 +92,30 @@
             .setMarshallDefaultValues(true)
             .build();
     private final PersistentResourceXMLDescription filesystemRealmParser_14_0 = builder(PathElement.pathElement(ElytronDescriptionConstants.FILESYSTEM_REALM), null)
-            .addAttributes(FileSystemRealmDefinition.ATTRIBUTES)
+            .addAttributes(FileSystemRealmDefinition.PATH)
+            .addAttribute(FileSystemRealmDefinition.RELATIVE_TO)
+            .addAttribute(FileSystemRealmDefinition.LEVELS)
+            .addAttribute(FileSystemRealmDefinition.ENCODED)
+            .addAttribute(FileSystemRealmDefinition.HASH_ENCODING)
+            .addAttribute(FileSystemRealmDefinition.HASH_CHARSET)
+            .build();
+    private final PersistentResourceXMLDescription filesystemRealmParser_15_0 = builder(PathElement.pathElement(ElytronDescriptionConstants.FILESYSTEM_REALM), null)
```

Comment:
I don't think this 15_0 version is needed.

Reply:
Removed